src/styles/shared/_icons.scss

```diff
@@ -19,6 +19,7 @@
 
 @mixin color-icon($value, $hue: base) {
   svg {
+    // have to get rid of this
     fill: color($value, $hue);
```

Comment:
What is keeping this one around?

Reply:
nothing! removed since it's not being used